docs/markdown/releasenotes.md

```diff
@@ -130,30 +160,34 @@
 * (IDETECT-2541) Resolved an issue that caused the CLANG detector to fail with "Unable to execute any supported package manager" when run with a non-English locale on an alpine system.
 
 ## Version 7.0.0
+
 ### New features
+
 * Added scripts detect7.sh and detect7.ps1 for invoking [solution_name] 7.x.x. detect.sh and detect.ps1 will (by default) continue to invoke the latest [solution_name] 6 version.
 * Added support for Yarn workspaces.
 * Added support for the dependency graph SBT plugin. Resolution cache generation is no longer a requirement of the SBT detector.
 * Added the properties [detect.excluded.directories](properties/configuration/paths.md#detect-excluded-directories-advanced),
-[detect.excluded.directories.defaults.disabled](properties/configuration/paths.md#detect-excluded-directories-defaults-disabled-advanced), and [detect.excluded.directories.search.depth](properties/configuration/signature-scanner.md#detect-excluded-directories-search-depth) to handle exclusions for detector search and signature scanning.
+  [detect.excluded.directories.defaults.disabled](properties/configuration/paths.md#detect-excluded-directories-defaults-disabled-advanced),
+  and [detect.excluded.directories.search.depth](properties/configuration/signature-scanner.md#detect-excluded-directories-search-depth) to handle exclusions for detector search and signature scanning.
 * Added ability to specify excluded directory paths using [glob patterns](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/FileSystem.html#getPathMatcher(java.lang.String)).
-* Added properties [detect.lerna.excluded.packages](properties/detectors/lerna.md#lerna-packages-excluded-advanced) and [detect.lerna.included.packages](properties/detectors/lerna.md#lerna-packages-included-advanced) to exclude and
-include specific Lerna packages.
+* Added properties [detect.lerna.excluded.packages](properties/detectors/lerna.md#lerna-packages-excluded-advanced) and [detect.lerna.included.packages](properties/detectors/lerna.md#lerna-packages-included-advanced) to exclude and include
```

Comment:
Change to read: " Added properties [detect.lerna.excluded.packages](properties/detectors/lerna.md#lerna-packages-excluded-advanced) and [detect.lerna.included.packages](properties/detectors/lerna.md#lerna-packages-included-advanced) to exclude or include specific Lerna packages."